clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -41,6 +41,9 @@
                                                        + "discover the full cluster membership (which may change dynamically), this list need not contain the full set of "
                                                        + "servers (you may want more than one, though, in case a server is down).";
 
+    public static final String CLIENT_DNS_LOOKUP = "client.dns.lookup";
+    public static final String CLIENT_DNS_LOOKUP_DOC = "Drives whether the client performs DNS lookups on entries of bootstrap.servers";
```

Comment:
I would phrase this as "To enable client DNS lookups on bootstrap.servers"

Reply:
changed.